library/tests/src/com/google/maps/android/BiMultiMapTest.java

```diff
@@ -0,0 +1,44 @@
+package com.google.maps.android;
+
+import junit.framework.TestCase;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class BiMultiMapTest extends TestCase {
+
+    private BiMultiMap<String> mMap = new BiMultiMap<>();
+
+    public void testSingle() {
+        String key = "foo";
+        String value = "bar";
+        mMap.clear();
+        mMap.put(key, value);
+        assertEquals(mMap.size(), 1);
```

Comment:
assert params should be expected first, then actual. This matters because a failing test shows a log statement like "Expected X, Got Y".